sys-libs/libxcrypt/libxcrypt-4.4.19.ebuild

```diff
@@ -136,8 +187,8 @@ multilib_src_install() {
 						static_libs=( "${ED}"/$(get_xclibdir)/*.a )
 
 						if [[ -n ${static_libs[*]} ]]; then
-							dodir "/usr/$(get_xclibdir)"
-							mv "${static_libs[@]}" "${D}/usr/$(get_xclibdir)" \
+							dodir "$(get_xclibdir)"
+							mv "${static_libs[@]}" "${D}$(get_xclibdir)" \
```

Comment:
Rolling back to D or ED should be fine with the suggested changes

Reply:
FWIW: I think since this was started, the ebuild properly respects ${EPREFIX} consistently throughout which might make some of our discussions a bit easier.